### PR TITLE
Added Votes in header 

### PR DESCRIPTION
**Pull request by vaibhavchellani**
Added votes in header so that they can be accessed via context in the app .

### CODE REVIEW

File: rpc/core/types/responses.go [hunk 1]
```diff
@@ -158,6 +158,7 @@ type ResultBroadcastTxCommit struct {
 	DeliverTx abci.ResponseDeliverTx `json:"deliver_tx"`
 	Hash      cmn.HexBytes           `json:"hash"`
 	Height    int64                  `json:"height"`
+	Data		[]byte					`json:"data"`
```
Comment (jdkanani) [inline rpc/core/types/responses.go]: Run `go fmt`

File: consensus/state.go [hunk 2]
```diff
@@ -10,7 +10,7 @@ import (
 	"sync"
 	"time"
 
-	fail "github.com/ebuchman/fail-test"
+	"github.com/ebuchman/fail-test"
```
Comment (jdkanani) [inline consensus/state.go]: What changed?

Comment (jdkanani) [reply consensus/state.go]: @vaibhavchellani

File: types/tx.go [hunk 3]
```diff
@@ -18,6 +18,7 @@ type Tx []byte
 
 // Hash computes the TMHASH hash of the wire encoded transaction.
 func (tx Tx) Hash() []byte {
+	fmt.Sprintf("Tx{%X} xoxo", []byte(tx))
```
Comment (jdkanani) [inline types/tx.go]: Please remove log from here (?)

File: consensus/state.go (hunk 2 above)
Comment (vaibhavchellani) [reply consensus/state.go]: it picks it up directly from package no need to use

File: types/tx.go (hunk 3 above)
Comment (vaibhavchellani) [reply types/tx.go]: done

**Pull request merged**